src/test/javascript/spec/component/shared/button.component.spec.ts

```diff
@@ -101,63 +100,63 @@ describe('ButtonComponent', () => {
         fixture.detectChanges();
 
         const title = getTitle();
-        expect(title).not.to.exist;
+        expect(title).toBe(null);
 
         const icon = getIcon();
-        expect(icon).to.exist;
+        expect(icon).not.toBe(null);
 
         const loadingIcon = getLoading();
-        expect(loadingIcon).not.to.exist;
+        expect(loadingIcon).toBe(null);
     });
 
-    it('should disable complete button if disabled is set', async () => {
+    it('should disable complete button if disabled is set', fakeAsync(() => {
         comp.title = 'artemisApp.title.test';
         comp.icon = 'redo';
         comp.disabled = true;
 
         fixture.detectChanges();
 
         const button = getButton();
-        expect(button).to.exist;
-        expect(button.disabled).to.be.true;
+        expect(button).not.toBe(null);
+        expect(button.disabled).toBe(true);
         button.click();
-        await fixture.whenStable();
+        tick();
 
-        expect(clickSpy).to.not.have.been.called;
-    });
+        expect(clickSpy).toHaveBeenCalledTimes(0);
```

Comment:
```suggestion
        expect(clickSpy).not.toHaveBeenCalled();
```
As by out guidelines

Reply:
I've missed that, thanks